src/lightning/pytorch/strategies/xla.py

```diff
@@ -211,13 +211,10 @@ def reduce(
 
     def setup_distributed(self) -> None:
         self._launched = True
-        self.set_world_ranks()
         rank_zero_only.rank = self.global_rank
 
     def set_world_ranks(self) -> None:
```

Comment:
The accelerator connector will call this method. But we don't want the XLA strategy setting the rank (yet). Since XLAStrategy inherits from DDPStrategy, we need to leave it empty here.